test/command-line.js

```diff
@@ -40,4 +40,49 @@ describe('command line', function () {
       return done();
     });
   });
+  var watchProc;
+  var stdout = '';
+  it('jade --no-debug --client --name-after-file --watch input-file.jade (pass 1)', function (done) {
+    fs.writeFileSync(__dirname + '/temp/input-file.jade', '.foo bar');
+    fs.writeFileSync(__dirname + '/temp/input-file.js', 'throw new Error("output not written (pass 1)");');
+    var args = [__dirname + '/../bin/jade.js', '--no-debug', '--client', '--name-after-file', '--watch', 'input-file.jade']
+    watchProc = cp.spawn('node', args,  {
+      cwd: __dirname + '/temp'
+    });
+
+    watchProc
+      .on('error', done)
+      .stdout.setEncoding('utf8');
+    watchProc.stdout.on('data', function(buf) {
+        stdout += buf;
+        if (/.*rendered.*/.test(stdout)) {
+          stdout = '';
+          var template = Function('', fs.readFileSync(__dirname + '/temp/input-file.js', 'utf8') + ';return inputFileTemplate;')();
+          console.log(template())
```

Comment:
Should probably remove any logging before you merge.  Funny thing is: I noticed it in the Travis CI output before I spotted it here!